connectors/data.json

```diff
@@ -0,0 +1,44 @@
+[
+    {
+        "name": "code-snippet",
+        "label": "Code Snippet",
+        "category": "utilities",
+        "description": "Execute code snippets as part of your Playbooks",
+        "publisher": null,
+        "operation_roles": [],
+        "configurations": [
+            {
+                "id": 31,
+                "config_id": "6452db36-dc11-44a1-9a7f-231a9b569ef5",
+                "status": 1,
+                "name": "Default",
+                "default": true,
+                "config": {
+                    "allow_imports": true
+                },
+                "remote_status": [],
+                "health_status": [],
+                "connector": 4,
+                "agent": "f8be236bf806712a32ea688b654f15b8",
+                "teams": []
+            }
+        ],
+        "dataImports": [],
+        "install_mode": "rpm",
+        "ingestionPlaybooks": {
+            "collections": [],
+            "exported_tags": []
+        }
+    },
+    {
+        "name": "fortinet-fortimanager_dev",
```

Comment:
> @fortijames Please ensure that the released version of the connector is added. Instead of "fortinet-fortimanager_dev", please add "fortinet-fortimanager".

This one is pending a connector that works with the solution pack. You just created the one we will be using. 
https://github.com/fortinet-fortisoar/connector-fortinet-fortimanager-json-rpc

My thinking here was to get things rolling so we have something started and then push updates to fix the dev connector. Anyone that uses this solution pack as a pre-release would need to download the dev connector and I can put that in the setup docs. I mentioned this in email as a step required before publishing to the Content Hub. 

In short. I expect this to be changed in the next PR and did not expect it to stop the merge of the rest of the SP.